src/net_processing.cpp

```diff
@@ -246,6 +246,7 @@ class PeerManagerImpl final : public PeerManager
 
     /** Implement PeerManager */
     void CheckForStaleTipAndEvictPeers() override;
+    bool FetchBlock(const NodeId nodeid, const CBlockIndex* pindex) override;
```

Comment:
No need to make the pass-by-value `NodeId nodeid` const.

Style guide is to use snake_case and not hungarian notation for parameter names:

```suggestion
    bool FetchBlock(const NodeId id, const CBlockIndex* index) override;
```